snappy_pipeline/workflows/somatic_variant_filtration/__init__.py

```diff
@@ -158,10 +158,10 @@ def _get_log_file(self, action):
 
         if action == "write_panel":
             # TODO: Possible bug. It should still yield dict-like format, no?
-            return (
-                "work/{mapper}.eb_filter.panel_of_normals/log/"
+            return {
```

Comment:
To keep it consistent, I would suggest removing the `return` and `yield` instead:
```
yield "log", "work/{mapper}.eb_filter.panel_of_normals/log/{mapper}.eb_filter.panel_of_normals.log"
```

...you will have to adjust the Snakemake file:
```
rule somatic_variant_filtration_eb_filter_write_panel:
    input:
       ...
    log:
        **wf.get_log_file("eb_filter", "write_panel"),
    run:
       ...
```

Reply:
...actually, it seems like the wrapper expects `snakemake.log`. 
Can you confirm that? If yes, this looks like a bigger ticket.

Reply:
There is no wrapper for the `write_panel` action: you can see in the [Snakefile](https://github.com/bihealth/snappy-pipeline/blob/bea1323bec554005b32466aff22bd2a435201346/snappy_pipeline/workflows/somatic_variant_filtration/Snakefile#L92) that the action is actually carried out using `run` rather than `wrapper`. And the [`write_panel_of_normals_file` function](https://github.com/bihealth/snappy-pipeline/blob/bea1323bec554005b32466aff22bd2a435201346/snappy_pipeline/workflows/somatic_variant_filtration/__init__.py#L346) doesn't create a log.

I have made the changes so that the `Snakefile` & `__init__.py` are in sync, and I removed a few comments about possible bugs, as I believe that the code is now correct.